src/SQLite.jl

```diff
@@ -275,107 +304,62 @@ function transaction(f::Function, db)
     finally
         # savepoints are not released on rollback
         commit(db, name)
-        execute(db,"PRAGMA synchronous = ON")
+        execute!(db,"PRAGMA synchronous = ON;")
     end
 end
 
-# commit a transaction or savepoint (if name is given)
-commit(db) = execute(db, "COMMIT TRANSACTION;")
-commit(db, name) = execute(db, "RELEASE SAVEPOINT $(name);")
+"commit a transaction or named savepoint"
+commit(db) = execute!(db, "COMMIT TRANSACTION;")
+"commit a transaction or named savepoint"
+commit(db, name) = execute!(db, "RELEASE SAVEPOINT $(name);")
 
-# rollback transaction or savepoint (if name is given)
-rollback(db) = execute(db, "ROLLBACK TRANSACTION;")
-rollback(db, name) = execute(db, "ROLLBACK TRANSACTION TO SAVEPOINT $(name);")
+"rollback transaction or named savepoint"
+rollback(db) = execute!(db, "ROLLBACK TRANSACTION;")
+"rollback transaction or named savepoint"
+rollback(db, name) = execute!(db, "ROLLBACK TRANSACTION TO SAVEPOINT $(name);")
 
-function droptable(db::SQLiteDB,table::AbstractString;ifexists::Bool=false)
+"drop the SQLite table `table` from the database `db`; `ifexists=true` will not return an error if `table` doesn't exist"
```

Comment:
Maybe re-word this to `\`ifexists=true\` will prevent an error being thrown if \`table\` doesn't exist"`?